src/OrchardCore.Modules/OrchardCore.Contents/Services/DefaultContentsAdminListFilter.cs

```diff
@@ -81,17 +78,15 @@ public async Task FilterAsync(ContentOptionsViewModel model, IQuery<ContentItem>
                 {
                     if (ctd.GetSettings<ContentTypeSettings>().Listable)
                     {
-                        var authorized = await _authorizationService.AuthorizeAsync(_httpContextAccessor.HttpContext.User, Permissions.EditContent, await _contentManager.NewAsync(ctd.Name));
+                        var authorized = await _authorizationService.AuthorizeAsync(_httpContextAccessor.HttpContext.User, Permissions.EditContent, new ContentItem { ContentType = ctd.Name });
```

Comment:
@Skrypt I don't think this is a good / necessary change?

Reply:
I think it is appropriate. Here we fake a content item only and it is not necessary to use the contentmanager to do so. In fact the contentmanager will trigger the events which here we don't want to since it's just a fake content item.

Reply:
In fact here this is the reason why content items we're not displaying properly. It was never evaluating if the current user has the EditContent permission globally on each content type. This is why the ListableContentType dictionary was always empty. The ContentTypeAuthorizationHandler was always returning false because it was expecting to receive a real ContentItem with a ContentItem.Owner != null. This is why I added a else statement in the handler. Here if you say that the contentManager was necessary because we wanted to throw the events on the CreateAsync method then I made a mistake but I don't think so since we should only be listing content items in that list.

So faking the ContentItem by using a new ContentItem instanciation here makes sense to me because it will be faster than using the ContentManager to create a fake ContentItem and throw it's events ...

Reply:
Let me know if it breaks something I didn't thought of.

Reply:
@sebastienros Here we are debating on if we should keep using the ContentManager to create a fake content item to evaluate. In case someone would have used the events to extend something when evaluating permissions when listing the content list ?!? Not sure this was a good call to use the ContentManager here so I removed it.